docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,103 @@
+
+Ollama could run the iGPU 780M of AMD Ryzen CPU at Linux base on ROCm. There only has a little extra settings than Radeon dGPU like RX7000 series.
+
+## Keys for usage
+- Ryzen 7000s/8000s CPU with iGPU 780M
+- amdgpu driver and rocm6.0
+- Linux OS is required (Windows and WSL2 are not supported)
+- BIOS must be set to enable the iGPU and dedicate > 1GB RAM to VRAM
+- HSA_OVERRIDE_GFX_VERSION="11.0.0" is set (extral setting for AMD iGPU-780M)
+
+## Prerequisites
+0. Set UMA for iGPU in BIOS. (at least >1GB, recommend to >8GB for Llama3:8b q4_0 model size is 4.7GB)
+1. Install GPU Driver and ROCm
+	Refer to
+	- [AMD ROCm™ documentation — ROCm Documentation](https://rocmdocs.amd.com/en/latest/)
+  	- [AMD ROCm™ Quick start installion](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/quick-start.html#rocm-install-quick)
+
+2. Install Ollama
+	
+ 	*`curl -fsSL https://ollama.com/install.sh | sh`*
+
+## Steps
+The iGPU is not detected by Ollama at default. We need extra steps to enable it.
+1. Stop the ollama.service
+   
+	`sudo systemctl stop ollama.service`
```

Comment:
If we take out the manual run as I suggest below, this step can be removed and start directly with editing the systemd config, since in the next step you do `sudo system restart ollama.service`

Reply:
Let's remain stop->edit->restart full flow to guide user to know the understand more about the service options even though some steps could be omitted.